rest_framework_docs/api_endpoint.py

```diff
@@ -122,7 +122,8 @@ def __get_serializer_fields__(self, serializer):
                     "type": str(field.__class__.__name__),
                     "sub_fields": sub_fields,
                     "required": field.required,
-                    "to_many_relation": to_many_relation
+                    "to_many_relation": to_many_relation,
+                    "max_length": (field.__dict__.get('max_length', '-') if field.__dict__.get('max_length', '-') != None else '-')
```

Comment:
change:

`if field.__dict__.get('max_length', '-') != None else '-')`

to

`if field.__dict__.get('max_length', '-') is not None else '-')`

Reply:
Committed the changes for flake8 requirements